src/components/panel/panel.spec.js

```diff
@@ -1439,6 +1469,7 @@ describe('$mdPanel', function() {
     // through the tests.
     $rootScope.$apply();
     $material.flushOutstandingAnimations();
+    return panelRef;
```

Comment:
You don't need to return panelRef. It is set in each of the if cases above, which means you can just use it.